go/items.json

```diff
@@ -0,0 +1 @@
+{"items":[{"name":"jacket","category":"fashion","image":""},{"name":"jacket","category":"fashion","image":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.jpg"}]}
```

Comment:
You can prevent this file from being added to your repository by adding it to the `.gitignore` file.